easybuild/easyconfigs/s/Score-P/Score-P-4.0-CrayCCE-18.07.eb

```diff
@@ -0,0 +1,69 @@
+# jg (CSCS)
+easyblock = 'ConfigureMake'
+name = 'Score-P'
+version = '4.0'
+
+
+homepage = 'http://www.score-p.org'
+description = """The Score-P measurement infrastructure is a highly scalable and
+ easy-to-use tool suite for profiling, event tracing, and online analysis of HPC
+ applications."""
+
+toolchain = {'name': 'CrayCCE', 'version': '18.07'}
+toolchainopts = {'pic': True}
+source_urls = ['http://www.vi-hps.org/upload/packages/scorep/']
+sources = ['scorep-%(version)s.tar.gz']
+checksums = [ 'f04478e0407d67eeb8c49c3c51d91e12' ]
+patches = [ '%(name)s-%(version)s-Cray.patch' ]
+
+dependencies = [
+    #('libiberty', '5.3.0-scorep', '', True),
```

Comment:
Could you please say something about the commented out lines (dependencies, configopts, etc..). If we don't need them we can delete the lines but if we need to keep theme as reminder it would be useful to have some kind of additional information.

Reply:
yes, they are just reminders.
I would prefer not to rebuild everything just because of minor comments and i need this PR to build other PRs so i will keep a note for future builds.
Do you mind approving this one ?

Reply:
If you are only changing comments you can skip the testing